.github/workflows/gh-ci.yaml

```diff
@@ -0,0 +1,70 @@
+name: GH Actions CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+concurrency:
+  group: "${{ github.ref }}-${{ github.head_ref }}-${{ github.workflow }}"
+  cancel-in-progress: true
+
+defaults:
+  run:
+    shell: bash -l {0}
+
+jobs:
+  unittests:
+    if: "github.repository == 'MDAnalysis/panedr'"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, ]
+        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10"]
+        include:
+          - os: windows-latest
```

Comment:
I forgot we set up griddataformats to check this way. You might want to set these up to be 3.10, doesn't matter much but then you're testing "latest on each OS"

Reply:
You could also do lowest and highest on windows and mac os. I *think* it would be
```
            python-version: ["3.6", "3.10"]
```
However, I'd be more than happy to just get CI going with highest and then tweak.

Reply:
I changed this to 3.10 for now, once CI is working I'll add 3.6 as well